src/components/aeAccountBackground/aeAccountBackground.md

```diff
@@ -0,0 +1,32 @@
+```js
+<ae-account-background type="exciting">
+  <span style="color: white">A flex box based container with rounded corners</span>
+  <ae-button>Button</ae-button>
+</ae-account-background>
+```
+
+```js
+const BN = require('bn.js')
+let identity = {
+  name: 'My identity',
+	address: '0x35d8830ea35e6Df033eEdb6d5045334A4e34f9f9',
+	balance: new BN('1337000000000000000', 10)
+}
```

Comment:
Here and in the example below are missed space in indention. Also, I prefer to use `const` instead of `let`, if a variable is not going to be reassigned.